src/components/transaction/TransactionTime.tsx

```diff
@@ -13,12 +13,12 @@ const TransactionTime: React.FC<TransactionTimeProps> = ({
 }: TransactionTimeProps) => (
   <span className="transaction-time-details-row">
     <div>
-      <DateRangeIcon style={{ color: '#7698A9', fontSize: 20 }} />
-      <span>{moment.unix(block_time || 0).format('MM-DD-YYYY')}</span>
+      <Icon icon={clockIcon} style={{ color: '#7698A9', fontSize: 18 }} />
+      <span>{moment.unix(block_time || 0).format('HH:mm:ss')}</span>
     </div>
     <div>
-      <Icon icon={clockIcon} style={{ color: '#7698A9', fontSize: 18 }} />
-      <span>{moment.unix(block_time || 0).format('hh:mm:ss')}</span>
+      <DateRangeIcon style={{ color: '#7698A9', fontSize: 20 }} />
+      <span>{moment.unix(block_time || 0).format('MM-DD-YYYY')}</span>
     </div>
   </span>
```

Comment:
I swapped the completion date and time around to fix the styling of this row: 

before: 
<img width="269" alt="image" src="https://user-images.githubusercontent.com/41127686/160241457-2a849873-160d-487b-a13d-4ea7a8f579f8.png">

after: 
<img width="232" alt="image" src="https://user-images.githubusercontent.com/41127686/160241487-ed0d4c7a-3180-4e23-a898-4c0b99f777c3.png">

(my CSS skills are questionable)